OutOfSchool/OutOfSchool.Encryption/Models/CASettings.cs

```diff
@@ -0,0 +1,20 @@
+namespace OutOfSchool.Encryption.Models;
+
+public class CASettings
+{
+    public string[] issuerCNs { get; set; }
+
+    public string address { get; set; }
+
+    public string ocspAccessPointAddress { get; set; }
+
+    public string ocspAccessPointPort { get; set; }
```

Comment:
("Stringly-typed" code) Why port (`ocspAccessPointPort` and `tspAddressPort`) has string type?

Reply:
@VadymLevkovskyi same as above - `C` library uses it as string.